pkg/hive/install.go

```diff
@@ -195,6 +214,7 @@ func (c *clusterManager) clusterDeploymentForInstall(doc *api.OpenShiftClusterDo
 			},
 			ControlPlaneConfig: hivev1.ControlPlaneConfigSpec{
 				APIServerIPOverride: doc.OpenShiftCluster.Properties.NetworkProfile.APIServerPrivateEndpointIP,
+				APIURLOverride:      "api-int." + installConfig.Config.ObjectMeta.Name + "." + installConfig.Config.BaseDomain + ":6443",
```

Comment:
Can't we do this:
```
	domain := m.doc.OpenShiftCluster.Properties.ClusterProfile.Domain
	if !strings.ContainsRune(domain, '.') {
		domain += "." + m.env.Domain()
	}
```

I would like to not read the graph if we can absolutely help it.